bin/bovine.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import os
+import yaml
+##########################
+# Read the Static Directory
+##########################
+
+
+class Inventory:
+  def __init__(self, root_directory):
+    # get the files in the root_directory
+    # self.
+    self.root_directory = root_directory
+    # import the yaml from the files that we found above
+  def list_files(self, root_directory):
```

Comment:
I would called this method 'get_file_names' (i.e. the style looks good, but the name could be better)

let's use the python pep8 standard: 
- methods like verb_noun_object_etc
- attributes like var_name

more at https://www.python.org/dev/peps/pep-0008/#package-and-module-names